docs/source/running-jobs/distributed-jobs.rst

```diff
@@ -49,5 +49,8 @@ SkyPilot exposes these environment variables that can be accessed in a task's ``
   You can retrieve the number of nodes by :code:`echo "$SKY_NODE_IPS" | wc -l`
   and the IP address of the third node by :code:`echo "$SKY_NODE_IPS" | sed -n
   3p`.
+
+  To manipulate these IP addresses, you can also store them to a file in the
```

Comment:
After rendering it's a bit hard to read. Can we make L49 and L53 nested bullets?